workspace-loader/webpack.common.js

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2018 Red Hat, Inc.
```

Comment:
it should be only 2018, in 2015 this loader didn't exist

Reply:
How it will be for pom.xml? 2018?

Reply:
Then we will have following
```
Copyright (c) 2018-2018 Red Hat, Inc.
```

Reply:
you may want to add
```
    <properties>
        <license_years>2018</license_years>
    </properties>
```
to have only 
```
    Copyright (c) 2018 Red Hat, Inc.
```

Reply:
Done.

Reply:
Let's decide. We have only 2018 or period 2018-2018.